pfsync status report

kprovost opened the pull request:
pfsync status report

wblock commented on 2018q4/pfsync.md:
```diff
@@ -0,0 +1,26 @@
+time.
+The buckets are independently locked, allowing multiple cores to proceed at once.
+
+The number of buckets is tunable, but defaults to 2 x number of cpus.
```
s/2 x/twice the/

wblock commented on 2018q4/pfsync.md:
```diff
@@ -0,0 +1,26 @@
+was loaded.
+
+This was improved by splitting the pfsync queues into buckets, based on the
+state id. This ensures that updates for a given connection always end up
```
s/id/ID/

wblock commented on 2018q4/pfsync.md:
```diff
@@ -0,0 +1,26 @@
+
+This was improved by splitting the pfsync queues into buckets, based on the
+state id. This ensures that updates for a given connection always end up
+in the same bucket, which allows pfsync to still collapse multiple
```
s/which allows/allowing/

wblock commented on 2018q4/pfsync.md:
```diff
@@ -0,0 +1,26 @@
+The buckets are independently locked, allowing multiple cores to proceed at once.
+
+The number of buckets is tunable, but defaults to 2 x number of cpus.
+Benchmarking has shown improvement, depending on hardware and setup, from ~30%
```
Flip around for clarity and to remove the aside:
`` Benchmarking has shown improvement of 30 to 100% depending on hardware and setup. ``

kprovost commented:
I believe I've addressed all of the remarks. Thanks for the review.
I'm not sure if there's some magic button here to move forward, or if I've already done everything required.

trasz commented:
Hello.  Warren, is there anything more to do before this can be merged?

trasz commented:
Thanks for submission; merged.
